pkgs/top-level/all-packages.nix

```diff
@@ -21855,5 +21855,6 @@ with pkgs;
   yrd = callPackage ../tools/networking/yrd { };
 
   powershell = callPackage ../shells/powershell { };
+  pps-tools = callPackage ../development/libraries/pps-tools { };
```

Comment:
This is primarily a set of Linux tools, please move it to `pkgs/os-specific/linux/` and after `powertop` in `all-packages.nix`.